cxx/pclean/get_joint_relations.cc

```diff
@@ -0,0 +1,50 @@
+#include "pclean/get_joint_relations.hh"
+
+#include <map>
+#include <string>
+#include <utility>
+
+#include "distributions/get_distribution.hh"
+#include "emissions/get_emission.hh"
+
+std::map<std::string, std::pair<std::string, std::string>> JOINT_NAME_TO_PARTS =
+{
+  // TODO(thomaswc): Implement an int emission function to use with the
+  // Skellam distribution.
+  {"bool", {"bernoulli", "sometimes_bitflip"}},
+  {"categorical", {"categorical", "sometimes_categorical"}},
+  {"real", {"normal", "sometimes_gaussian"}},
```

Comment:
Why did you choose `sometimes_gaussian` instead of `gaussian` here? It isn't clear to me which is a better default assumption (`gaussian` seems more parsimonious for a noisy measurement, but for tabular data/modeling human error, maybe it's important to model the presence/absence of an error too?) I'd lean towards `gaussian` here, since it's simpler and probably easier to infer (as we stand up inference testing of the whole system), unless there's a reason to immediately use `sometimes_gaussian`.

Reply:
I picked sometimes_gaussian because it is a lot more flexible, and I think we are going to need that flexibility in practice.

Many real numbers come from measurements of semi-fixed things, and those real numbers are usually not going to be corrupted when reported by another source.  Think of the area of a country, or the height of a person, etc.